serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/db/Db.kt

```diff
@@ -0,0 +1,40 @@
+package xyz.room409.serif.serif_shared.db
+import xyz.room409.serif.serif_shared.db.SessionDb
+import xyz.room409.serif.serif_shared.db.*
+import com.squareup.sqldelight.db.SqlDriver
+
+expect class DriverFactory {
+    fun createDriver(): SqlDriver
+}
+
+fun saveSession(db : SessionDb?, username : String, access_token : String, transactionId : Long) {
+    if(db != null) {
```

Comment:
These could be made a bit more compact with the nullability operators

db?.sessionDbQueries?.insertSession(username, access_token, transactionId)

Reply:
That's slick.